uint8 left=0
uint8 right=1

uint8 foot
float64 footstep_x
float64 footstep_y
float64 footstep_z
float64 footstep_yaw
float64 footstep_height
float64 footstep_duration

float64 base_height
float64 base_speed
---
uint8 fail=0
uint8 success=1
uint8 joint_states_missing=2

uint8 status